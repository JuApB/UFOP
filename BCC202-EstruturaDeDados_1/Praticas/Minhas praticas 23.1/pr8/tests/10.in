10
24
Oliver 19 3
LuizHenrique 50 1
AnaJulia 89 2
Luana 69 2
MariaHeloisa 75 2
MariaCecilia 36 3
PedroLucas 30 1
Kaique 44 2
Ryan 56 3
Clarice 73 3
MariaLuiza 36 3
Clara 36 2
PedroMiguel 10 1
Eduardo 80 1
Gustavo 38 2
JoaoGabriel 78 3
Sophie 16 3
MariaJulia 61 1
Davi 76 1
Nathan 63 1
Rebeca 88 2
Lucca 63 2
Julia 30 3
Pietro 79 1
28
AnaLivia 75 3
Laura 60 3
Otavio 73 3
Beatriz 77 1
DaviLucca 50 1
Isabella 61 2
MariaVitoria 48 1
ArthurHenrique 26 1
Oliver 69 2
LuizOtavio 64 1
Clara 10 1
Kaique 59 1
Heloisa 71 1
Emilly 89 1
Lorenzo 80 2
Lavinia 85 1
Vitoria 20 1
Tomas 53 2
Alice 27 1
Anthony 71 1
Benjamin 66 3
MariaAlice 65 1
Antonia 92 1
Fernando 30 2
Eduarda 81 1
Mirella 17 3
MariaHelena 84 3
Gabrielly 48 2
22
CarlosEduardo 53 3
Alice 19 3
Larissa 100 1
Louise 40 2
Joana 25 1
Clara 23 3
Heloisa 42 1
Sarah 38 3
Stella 60 2
Erick 40 1
Rafael 82 3
Anthony 14 3
Isabelly 54 2
Henry 26 3
Valentina 100 3
Marina 33 1
Isis 38 1
Rodrigo 16 1
Emilly 41 3
LuizFelipe 78 2
Miguel 86 1
Heitor 17 3
10
Davi 51 2
Beatriz 20 1
Otavio 49 1
Guilherme 83 2
Danilo 95 2
Luan 68 3
Eloa 52 3
Thiago 30 1
Yuri 55 1
MariaVitoria 25 1
25
Louise 70 3
Eduarda 37 3
EnzoMiguel 32 3
MariaSophia 70 1
Benjamin 26 3
Beatriz 68 3
Noah 38 2
Sophie 15 2
Milena 11 3
MariaLuiza 19 1
DaviMiguel 98 2
Lucas 19 1
Lais 31 2
Miguel 19 2
Rafaela 24 2
Leonardo 30 2
Yago 50 1
Rodrigo 78 3
Otavio 41 2
Martin 59 3
Gabrielly 56 3
Daniel 32 1
Rafael 83 1
Bianca 99 3
Valentina 82 3
38
Mirella 16 2
Bruna 40 1
Isabelly 54 3
Isis 35 2
Pietra 26 1
AnaLivia 43 1
Murilo 12 3
Heloisa 30 3
Yago 22 1
Oliver 98 2
Samuel 47 3
Melissa 62 3
Andre 55 3
Emanuel 22 1
AnaCecilia 94 3
Bryan 49 2
Isadora 66 1
CarlosEduardo 55 3
Louise 44 3
Lavinia 67 1
Joao 88 2
Nicole 53 3
VictorHugo 94 3
Yuri 73 3
AnaVitoria 80 2
Amanda 65 3
Isaac 70 1
Augusto 88 1
Henrique 55 3
Theo 86 2
MariaValentina 29 2
Larissa 34 2
Martin 89 1
Danilo 52 2
Caio 40 3
Tomas 23 2
Olivia 46 2
LuizFelipe 60 2
27
Julia 78 1
Kaique 21 3
Mariah 71 2
Eduardo 91 2
Lais 66 2
AnaLiz 97 3
Isadora 96 3
Yuri 48 3
Maya 29 2
AnaLuiza 67 1
Daniel 29 1
Carolina 32 3
Felipe 31 2
Antonia 85 1
EnzoGabriel 44 3
Eloa 92 1
Pietra 86 2
Gael 30 3
Francisco 29 1
MariaVitoria 10 1
Ian 68 1
AnaCecilia 25 1
Yasmin 49 3
Yago 29 3
ArthurHenrique 81 2
Matheus 70 2
Giovanna 19 1
38
Liz 94 1
Mariana 72 2
Isabel 40 2
MariaHeloisa 37 1
Alice 57 3
Sophia 69 3
Luan 33 1
Victor 13 3
Olivia 97 1
DaviLucas 44 1
Lorena 51 2
Henrique 57 3
VictorHugo 47 3
MariaAlice 32 3
Benicio 38 1
AnaJulia 19 1
JoaoPedro 39 1
Matheus 25 2
ArthurHenrique 69 2
Melissa 41 3
Luna 72 2
Thiago 58 1
DaviMiguel 55 1
Gabrielly 85 2
Lavinia 76 1
PedroHenrique 98 3
AnaClara 78 1
Gustavo 16 3
ArthurMiguel 96 1
MariaSophia 57 2
Livia 59 2
Alicia 36 1
Lara 17 3
AnaLaura 25 3
Enrico 10 2
Louise 14 1
Fernando 27 2
Rodrigo 84 2
8
JoaoMiguel 40 3
MariaClara 33 3
ArthurHenrique 62 2
Mirella 15 2
Bianca 84 1
JoaoGabriel 38 2
Gustavo 67 2
Pedro 73 3
32
Emilly 69 3
Miguel 18 2
Oliver 59 3
AnaCecilia 21 2
MariaFlor 31 3
Theo 48 1
Benjamin 90 3
Lorena 38 1
MariaJulia 69 3
AnaJulia 54 1
JoaoLucas 54 2
Breno 72 1
Augusto 93 2
Tomas 43 1
Eloa 45 2
JoaoMiguel 26 3
Rafael 95 1
JoaoGabriel 83 1
DaviLuiz 12 3
Catarina 87 2
Cecilia 90 2
Fernanda 24 3
AnaBeatriz 72 3
Rodrigo 34 2
Valentina 36 1
Joana 91 3
Victor 71 3
MariaFernanda 68 3
Gustavo 47 1
Leonardo 28 2
Nicole 30 1
Vinicius 90 3
