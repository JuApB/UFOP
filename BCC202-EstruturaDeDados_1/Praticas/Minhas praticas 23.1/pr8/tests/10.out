Valentina 100 3
Bianca 99 3
AnaCecilia 94 3
AnaLiz 97 3
PedroHenrique 98 3
VictorHugo 94 3
Otavio 73 3
Sophie 16 3
Isadora 96 3
Valentina 82 3
Louise 70 3
Beatriz 68 3
Joana 91 3
Yuri 73 3
Melissa 62 3
Anthony 14 3
Theo 86 2
Luan 68 3
Eloa 52 3
Pietra 86 2
Rodrigo 78 3
Sophia 69 3
Gabrielly 56 3
AnaJulia 89 2
Isabella 61 2
Pedro 73 3
Benjamin 90 3
AnaBeatriz 72 3
MariaFernanda 68 3
Laura 60 3
Heloisa 30 3
Joao 88 2
AnaVitoria 80 2
Luana 69 2
Danilo 52 2
Isis 35 2
Ryan 56 3
EnzoGabriel 44 3
Gael 30 3
Davi 51 2
Matheus 70 2
MariaHeloisa 75 2
Clarice 73 3
Martin 59 3
Henrique 57 3
VictorHugo 47 3
Oliver 19 3
Mariana 72 2
Rafaela 24 2
Isabel 40 2
MariaSophia 57 2
Benjamin 26 3
Benjamin 66 3
MariaCecilia 36 3
Vinicius 90 3
Victor 71 3
CarlosEduardo 55 3
Oliver 59 3
Louise 44 3
Samuel 47 3
Andre 55 3
Murilo 12 3
Heitor 17 3
Isabelly 54 2
LuizFelipe 60 2
Larissa 100 1
Kaique 44 2
Anthony 71 1
Augusto 88 1
Larissa 34 2
Louise 40 2
Miguel 86 1
Tomas 23 2
Mirella 17 3
Yasmin 49 3
Kaique 21 3
Carolina 32 3
Guilherme 83 2
Yago 29 3
Antonia 85 1
Eloa 92 1
Maya 29 2
Thiago 30 1
AnaLuiza 67 1
Mariah 71 2
ArthurHenrique 81 2
MariaLuiza 36 3
Alice 57 3
Melissa 41 3
Eduarda 37 3
Lara 17 3
Luna 72 2
Milena 11 3
Gustavo 16 3
Victor 13 3
Lais 31 2
ArthurHenrique 69 2
Pietro 79 1
Miguel 19 2
Clara 36 2
ArthurMiguel 96 1
Otavio 41 2
Heloisa 71 1
Enrico 10 2
Matheus 25 2
Olivia 97 1
JoaoMiguel 40 3
MariaClara 33 3
EnzoMiguel 32 3
JoaoGabriel 78 3
MariaHelena 84 3
Amanda 65 3
Emilly 69 3
JoaoMiguel 26 3
Nicole 53 3
Caio 40 3
Henrique 55 3
Clara 23 3
Oliver 98 2
Catarina 87 2
Sarah 38 3
Stella 60 2
Isabelly 54 3
LuizFelipe 78 2
Bryan 49 2
Lorenzo 80 2
Olivia 46 2
Lavinia 85 1
Lavinia 67 1
Tomas 53 2
Pietra 26 1
Bruna 40 1
AnaLivia 43 1
Heloisa 42 1
Mirella 16 2
Marina 33 1
Isaac 70 1
Emanuel 22 1
Isis 38 1
Rodrigo 16 1
MariaValentina 29 2
Martin 89 1
Antonia 92 1
Erick 40 1
Isadora 66 1
Yago 22 1
MariaJulia 61 1
Joana 25 1
Fernando 30 2
Davi 76 1
Yuri 48 3
LuizHenrique 50 1
Danilo 95 2
Gabrielly 48 2
Eduardo 91 2
MariaVitoria 48 1
Ian 68 1
MariaVitoria 25 1
Lais 66 2
ArthurHenrique 26 1
Nathan 63 1
Otavio 49 1
Julia 78 1
Vitoria 20 1
Yuri 55 1
Francisco 29 1
MariaVitoria 10 1
Beatriz 20 1
AnaCecilia 25 1
Daniel 29 1
Beatriz 77 1
Alice 27 1
Felipe 31 2
Giovanna 19 1
Rebeca 88 2
DaviLucca 50 1
Julia 30 3
Lucca 63 2
AnaLaura 25 3
LuizOtavio 64 1
MariaAlice 32 3
Noah 38 2
Rodrigo 84 2
MariaAlice 65 1
Lorena 51 2
Leonardo 30 2
Gabrielly 85 2
Clara 10 1
Livia 59 2
MariaHeloisa 37 1
DaviMiguel 98 2
Kaique 59 1
Fernando 27 2
DaviLucas 44 1
Eduarda 81 1
Thiago 58 1
DaviMiguel 55 1
Liz 94 1
Lavinia 76 1
Sophie 15 2
AnaClara 78 1
Luan 33 1
Yago 50 1
MariaSophia 70 1
Benicio 38 1
Alicia 36 1
AnaJulia 19 1
PedroMiguel 10 1
JoaoPedro 39 1
Louise 14 1
MariaLuiza 19 1
Daniel 32 1
Emilly 89 1
Rafael 83 1
Gustavo 67 2
Lucas 19 1
Bianca 84 1
JoaoGabriel 38 2
ArthurHenrique 62 2
Mirella 15 2
CarlosEduardo 53 3
AnaLivia 75 3
Rafael 82 3
Eduardo 80 1
Emilly 41 3
Alice 19 3
MariaJulia 69 3
Oliver 69 2
DaviLuiz 12 3
Cecilia 90 2
MariaFlor 31 3
Rodrigo 34 2
Gustavo 38 2
Henry 26 3
Eloa 45 2
AnaCecilia 21 2
Rafael 95 1
JoaoGabriel 83 1
Miguel 18 2
PedroLucas 30 1
AnaJulia 54 1
Fernanda 24 3
JoaoLucas 54 2
Breno 72 1
Valentina 36 1
Augusto 93 2
Theo 48 1
Tomas 43 1
Gustavo 47 1
Leonardo 28 2
Nicole 30 1
Lorena 38 1
